### PR TITLE
DOCS-4616 Managing App Data update

### PR DESCRIPTION
**Pull request by george-hoffer**
https://www.mulesoft.org/jira/browse/DOCS-4616

### CODE REVIEW

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc [hunk 1]
```diff
@@ -19,26 +19,27 @@ Your application can also store custom integration data as well, within the limi
 [NOTE]
 ====
 * Object stores are handled in the way described in this document only for applications deployed to CloudHub workers. For applications that you deploy via the Runtime Manager to local servers, you must handle the storage of data in your own repository. See xref:deployment-strategies.adoc[Deployment Strategies] for more details.
-* All data in an object store is handled by servers in the *US-East zone*, regardless of what zone your application is deployed to. This may impact performance.
+* All data in Object Store v1 is handled by servers in the *US-East zone*, regardless of what zone your application is deployed to. This may impact performance.
+* Object Store v2 provides expanded cloud object store functionality for use with Mule 3.8.5 and later and for Mule 4.
```
Comment (steveCluster) [inline modules/ROOT/pages/managing-application-data-with-object-stores.adoc]: expanded how?  should we provide a link here?

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc [hunk 2]
```diff
@@ -49,19 +50,28 @@ To make properties secure so that they're not visible using the Runtime Manager
...
 * 1 GB of data per application
 
+Object Store v2 has these limits:
+
+* Works with Mule 3.8.5 and later, and Mule 4
+* Unlimited number of keys per application
+* Values can be up to 10 MB
+* Trial level 10 TPS, premium level 100 TPS
+* Keys persist for up to 30 days
+* Available in 10 regions around the world
```
Comment (steveCluster) [inline modules/ROOT/pages/managing-application-data-with-object-stores.adoc]: Available in all commercially available Cloudhub regions.

Comment (george-hoffer) [reply modules/ROOT/pages/managing-application-data-with-object-stores.adoc]: Changed to:
Available in xref:object-store::osv2-faq.adoc[all commercially available CloudHub regions].

The FAQ's first section lists what regions are available.

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/managing-application-data-with-object-stores.adoc]: Added link and clarified. Bullet is now:
* xref:object-store::osv2-faq.adoc[Object Store v2] expands the number of cloud object store regions and object store capabilities for use with Mule 3.8.5 and later and for Mule 4.

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc [hunk 3]
```diff
@@ -49,19 +50,28 @@ To make properties secure so that they're not visible using the Runtime Manager
...
 * 1 GB of data per application
 
+Object Store v2 has these limits:
+
+* Works with Mule 3.8.5 and later, and Mule 4
+* Unlimited number of keys per application
+* Values can be up to 10 MB
+* Trial level 10 TPS, premium level 100 TPS
+* Keys persist for up to 30 days
+* Available in xref:object-store::osv2-faq.adoc[all commercially available CloudHub regions].
```
Comment (valkyrie69) [inline modules/ROOT/pages/managing-application-data-with-object-stores.adoc:69]: How is this a limitation? (line 69)

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc [hunk 4]
```diff
@@ -49,19 +50,28 @@ To make properties secure so that they're not visible using the Runtime Manager
...
+
 == Advanced Information
 
-Internally, Mule has several different types of object stores. Runtime Manager provides an implementation of the user defined object store. This is for runtime information about your application, such as those use cases mentioned above. If you wish to access this object store directly, it can be found internally in the Mule Registry under the `_defaultUserObjectStore`. The Object Store module and connectors use this store to store all their data by default.
+Internally, Mule has several different types of object stores. Runtime Manager provides Object Store v1 (for Mule 3) and Object Store v2 (for Mule 3.8.5 and later, and Mule 4). These object stores are for runtime information about your application, such as those use cases mentioned above. If you wish to access this object store directly, it can be found internally in the Mule Registry under the `_defaultUserObjectStore`. The Object Store module and connectors use this store to store all their data by default.
```
Comment (valkyrie69) [inline modules/ROOT/pages/managing-application-data-with-object-stores.adoc]: You can find this object store internally in the Mule Registry under the '_defaultUserObjectStore` if you want to access it directly. (something like that to get rid of passive construction)

**Review by valkyrie69 — APPROVED** [review]
Minor comments. I wasn't sure about the capitalization of object store. It seems inconsistent. I think it should only be capitalized when we are talking about the product. LC when we are just talking about an object store conceptually. But the capitalization here doesn't seem to follow that.

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc (hunk 3 above)
Comment (george-hoffer) [reply modules/ROOT/pages/managing-application-data-with-object-stores.adoc:69]: Deleted "limits"

**Pull request merged**